package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint --ignore-path .gitignore .",
     "pretest": "npm run lint",
-    "tdd": "NODE_ENV=test env $(cat test.env | xargs) mocha -R dot --recursive --watch",
+    "tdd": "NODE_ENV=test env $(cat test.env | xargs) mocha -R dot --recursive --watch --bail",
```

Comment:
Completely unrelated to this PR, but `env $(cat test.env | xargs)` is a very cool trick. I'm totally stealing that.

Reply:
it's pretty brittle. it doesn't support comments and multiline secrets but I hope to keep using it out as far as it'll gooooooo.

We try to use`cat example.env` but those limitations make it hard to keep using as a project grows.